app/controllers/api/v1/api_controller.rb

```diff
@@ -0,0 +1,16 @@
+class Api::V1::ApiController < ApplicationController
+
+  before_action :find_account
+
+  respond_to :json
+
+  private
+
+  def find_account
+    api_token = (request.headers['HTTP_AUTHORIZATION'] || '').gsub(/Bearer(\s*)/, '')
```

Comment:
Line is too long. [85/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.